src/media/README.md

```diff
@@ -0,0 +1,17 @@
+# Media
+
+A collection of tools for working with media, such as images and
+videos. 
+
+## ASCII
+
+### Images
+
+We have a tool which can convert images to ASCII in `images.py`.
```

Comment:
Write small example tests to show these work. (perhaps not for the webcam one as most servers won't have a webcam).

Reply:
Done for the image.